std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/DoubleRoundOp.java

```diff
@@ -0,0 +1,84 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import org.enso.polyglot.common_utils.Core_Math_Utils;
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
+import org.enso.table.data.column.storage.numeric.DoubleStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+import java.util.BitSet;
+
+/** An operation rounding floating-point numbers. */
+public class DoubleRoundOp extends TernaryMapOperation<Double, DoubleStorage> {
+
+    public DoubleRoundOp(String name) {
+        super(name);
+    }
+
+    @Override
+    public Storage<?> runTernaryMap(DoubleStorage storage, Object decimalPlacesObject, Object useBankersObject, MapOperationProblemBuilder problemBuilder) {
+        if (decimalPlacesObject == null || useBankersObject == null) {
+            return DoubleStorage.makeEmpty(storage.size());
+        }
+
+        if (!(decimalPlacesObject instanceof Long decimalPlaces)) {
+            throw new UnexpectedTypeException("a long.");
+        }
+        if (!(useBankersObject instanceof Boolean useBankers)) {
+            throw new UnexpectedTypeException("a boolean.");
+        }
+
```

Comment:
These checks are good, but we should make sure they are never exposed to the user. As documented in the header of this type:
```
 * <p>This is an internal error of the Table library and any time it is thrown indicates a bug.
 * Normally, the types should be checked before being passed to a vectorized operation.
```

Let's add some tests that ensure that:
```
Test.expect_panic_with (column.round use_bankers="string") Type_Error
Test.expect_panic_with (column.round decimal_places="string") Type_Error
```

Reply:
Done